### DISCUSSION TITLE
Welcome to requirements as code discussions!

### DISCUSSION THREAD

**Discussion by bertilmuth** [bertilmuth/requirementsascode#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drranzinger** [comment]
Requirements as code is the most awesome project on github, still.
But not for long, because mind as #cheese will soon surpass it in an #agile way with its open source #PersonalityVirtualization engine based on #AgileCheese.
